EIPS/eip-2681.md

```diff
@@ -37,6 +37,8 @@ This mode of replay protection is out of fashion since [EIP-155](./eip-155.md) i
 
 3. Most clients already consider the nonce field to be 64-bit, such as go-ethereum.
 
+4. The reason a transaction with nonce `2^64-1` is invalid, because after inclusion the account's nonce would exceed `2^64-1`.
```

Comment:
```suggestion
4. The reason a transaction with nonce `2^64-1` is invalid, because otherwise after inclusion the sender account's nonce would exceed `2^64-1`.
```